compiler/codegen/cg-symbol.cpp

```diff
@@ -238,7 +244,31 @@ void preparePrintLlvmIrForCodegen() {
   forv_Vec(FnSymbol, fn, gFnSymbols) {
     if (shouldLlvmPrintIrFn(fn)) {
       addCNameToPrintLlvmIr(fn->cname);
+      // mark Chapel symbol as found
+      llvmPrintIrNames[astr(fn->name)] = true;
```

Comment:
Does this just not handle overloading at all?

Reply:
I just tested as I didn't know the answer myself -- it prints disassembly of all overloads.